priv/templates/phoenix.gen.json/controller_test.exs

```diff
@@ -21,9 +21,9 @@ defmodule <%= module %>ControllerTest do
       "<%= k %>" => <%= singular %>.<%= k %><% end %>}
   end
 
-  test "does not show resource and instead throw error when id is nonexistent", %{conn: conn} do
+  test "renders page not found when id is nonexistent", %{conn: conn} do
```

Comment:
I made the message exactly the same as in `phoenix.gen.html` for this test. I can change it in the other place, or in both to something else.